src/lualib/declarations/math.d.ts

```diff
@@ -8,4 +8,7 @@ declare namespace math {
     function atan(y: number, x?: number): number;
 
     function atan2(y: number, x: number): number;
+
+    function ceil(x: number): number;
+    function floor(x: number): number;
```

Comment:
Why not just `Math.*`?

Reply:
I know these methods do what we want, if we can have such extremely complicated functions like these as lualib functions I would rather not risk it and use the lowest level possible in lualib implementations.